src/ens/index.js

```diff
@@ -0,0 +1,21 @@
+const ENS = require('ethereum-ens')
+
+module.exports = {
+  /**
+   * Get the address of an ENS name.
+   *
+   * @param  {string} name
+   * @param  {object} web3
+   * @param  {string} [registryAddress=null]
+   * @return {string}
```

Comment:
My linter will freak out if I mark a function `async` and return a promise (without otherwise using `await`), but the docblock should be updated regardless